.github/ISSUE_TEMPLATE/📲contribute-an-azd-template-.md

```diff
@@ -8,56 +9,32 @@ assignees: nitya, savannahostrowski
 ---
 
 ## New azd-template Submission
```

Comment:
If we are eliminating the "submit a requested template' issue template, perhaps we should include a field here that allows them to specify any issue this might close?